.gitignore

```diff
@@ -0,0 +1 @@
+/.idea/
```

Comment:
Let's leave this out of the repo's gitignore, this can be [ignored locally (globally & per-repo)](https://help.github.com/articles/ignoring-files/) for developers who are using Idea's editors and others.